modules/services/utils/src/main/java/au/com/shiftyjelly/pocketcasts/utils/featureflag/Feature.kt

```diff
@@ -179,6 +179,14 @@ enum class Feature(
         hasFirebaseRemoteFlag = false,
         hasDevToggle = false,
     ),
+    REFERRALS(
+        key = "referrals",
+        title = "Referrals",
+        defaultValue = BuildConfig.DEBUG,
+        tier = FeatureTier.Free,
```

Comment:
Although gift icon is shown only to `Paid` accounts, the receiving account is free, so the minimum feature tier is set to `Free` instead of `Plus`.